saleor/checkout/tests/test_checkout.py

```diff
@@ -1491,3 +1491,26 @@ def test_cancel_active_payments(checkout_with_payments):
 
     # then
     assert checkout.payments.filter(is_active=True).count() == 0
+
+
+def test_create_order_with_variant_tracking_false(
+    checkout, customer_user, variant_without_inventory_tracking
+):
+    variant = variant_without_inventory_tracking
+    add_variant_to_checkout(checkout, variant, 10, check_quantity=False)
+    checkout.user = customer_user
+    checkout.billing_address = customer_user.default_billing_address
+    checkout.shipping_address = customer_user.default_billing_address
+    checkout.save()
```

Comment:
```suggestion
    checkout.user = customer_user
    checkout.billing_address = customer_user.default_billing_address
    checkout.shipping_address = customer_user.default_billing_address
    checkout.save()
    add_variant_to_checkout(checkout, variant, 10, check_quantity=False)
```
We should first set checkout addresses then assign variant to checkout because the possibility to assign variant to checkout is related to address.

Reply:
@fowczarek Thanks for the suggestion.
But during `add_variant_to_checkout` , no address check happens.  So i don't see anything specific check in your version of code. Let me know if i am missing on some scenarios.

Also the same flow is used in `test_create_order_insufficient_stock`.  So i am assuming the flow is correct.

Reply:
`add_variant_to_checkout` call `check_variant_in_stock` and this use `checkout.get_country()` 
Addreses are used by  `checkout.get_country()`.

In `test_create_order_insufficient_stock` we use different order to check if `prepare_order_data` raises an error when one of a product becomes unavailable after is added to the checkout.

Reply:
@fowczarek Got your point and this should be the case of a general product where inventory_tracking is set to True.

But i has one doubt. Here we are checking for variant for which inventory_tracking is set to false.

Now when we are not not tracking inventory, skipping the stock availability check in `check_stock_quantity`, then does this suggestion really affect the flow ?